tests/python/bls_test_template.py

```diff
@@ -0,0 +1,99 @@
+import taichi as ti
+
+
+def bls_test_template(dim,
```

Comment:
This function is the old `_test_bls_stencil` in `test_bls.py`. I moved it here so that I can reuse it in the `benchmark_bls.py` file.